<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QMainWindow" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数字图像处理系统（韩敬康，尹珩宇）</string>
  </property>
  <property name="windowIcon">
   <iconset resource="allicons.qrc">
    <normaloff>:/new/prefix1/C:/Users/Han JingKang/Desktop/ico.jpg</normaloff>:/new/prefix1/C:/Users/Han JingKang/Desktop/ico.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>161</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(47, 47, 47);</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>696</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="grayBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="histBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="smoothBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sharpenBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="noiseBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shapeBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="scrollArea" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>671</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(68, 68, 68);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>551</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);
background-color: rgb(144, 144, 144);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>281</width>
      <height>731</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(140, 152, 255);
background-color: rgb(47, 47, 47);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55,55,55);</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>552</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="preBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enlargeBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/C:/Users/Han JingKang/Desktop/icon/++.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reduceBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spinBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mirrorBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cutBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>78</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="actiona"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFile">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actiona">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="allicons.qrc"/>
 </resources>
 <connections/>
</ui>
